### DISCUSSION TITLE
within project dashboard [Sep 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3901] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Sinwan Meynaud**  
#### Here is my swagger authentification page : https://app2tests.herokuapp.com/api/v1/db/meta/projects/p_w703vf6cu88ar8/swagger  
#  
> 🙋‍♂️ **Sinwan Meynaud**  
#### But I don't have authentication in it  
#  
> 🙋‍♂️ **Sinwan Meynaud**  
#### the endpoint auth-project-user-list works good  
#  
> 🙋‍♂️ **Sinwan Meynaud**  
#### but no way to have a successful request for signup or signin  
#  
> 💁 **o1lab**  
#### apis.nocodb.com  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1020436507435204638)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
